Cargo.toml

```diff
@@ -138,7 +138,7 @@ inventory = "0.1"
 maxminddb = { version = "0.13.0", optional = true }
 strip-ansi-escapes = { version = "0.1.0", optional = true }
 colored = "1.9"
-warp = { version = "0.1.20", optional = true }
+warp = { version = "0.1", git = "https://github.com/timberio/warp", branch = "0.1.x", default-features = false, optional = true }
```

Comment:
Does this need documenting why we're on a moving target git branch here?

Reply:
I can add something for that! Good idea!